### ISSUE TITLE
Failed to get data for similar features in chromatin accessibility

### ISSUE THREAD

**Issue by YingX97** [fabilab/cell_atlas_approximations_API#19]
Hi Fabio,

Q: what are 5 peaks similar to chr1:1347569-1347969 in human stomach

I have specified all the params, but still failed to fetch the data from the API
1) using Node.js
<img width="1020" alt="Screen Shot 2023-12-06 at 11 14 35 am" src="https://github.com/fabilab/cell_atlas_approximations_API/assets/98261770/c0183d87-e1fa-4c7a-a650-8268602f5c4b">

2) API url:
https://api.atlasapprox.org/v1/similar_features?organism=h_sapiens&organ=stomach&feature=chr1:1347569-1347969&number=5&measurement_type=chromatin_accessibility

{
"message": "Internal Server Error"
}

**Comment by iosonofabio** [comment]
Yep, noticed yesterday, I'll fix ASAP

**Comment by iosonofabio** [comment]
this has something to do with feature naming and lowercasing, investigating

**Comment by iosonofabio** [comment]
fixed, next release
